tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+requires =
+    tox-conda
+
+[testenv]
+conda_spec =
+    py38: {toxinidir}{/}requirements{/}locks{/}py38-linux-64.lock
+    py39: {toxinidir}{/}requirements{/}locks{/}py39-linux-64.lock
+    py310: {toxinidir}{/}requirements{/}locks{/}py310-linux-64.lock
+
+[testenv:py{38,39,310}-tests]
+description = Perform Iris unit, integration and system tests.
+usedevelop =
+    true
+commands =
+    pytest -n auto {posargs} {toxinidir}{/}lib{/}iris{/}tests
```

Comment:
I'm not clear from the documentation whether `{posargs}` is a special value to `tox`?

Reply:
See [here](https://tox.wiki/en/3.27.1/example/general.html#interactively-passing-positional-arguments)

so if you were to run:
```tox -e py310-tests --cov``

it will create the py3.10 environment, install a dev version of iris, then run:
```pytest -n -auo --cov ./lib/iris/tests```